packer/openstack.pkr.hcl

```diff
@@ -101,6 +101,11 @@ variable "floating_ip_network" {
   default = null
 }
 
+variable "manifest_output_path" {
+  type = string
+  default = null
```

Comment:
Can we not set a default for this, so we don't have to specify it in every environment? As it is, this PR will break CI.

Reply:
Updated.